docs/Behaviour - Server Side.md

```diff
@@ -1,44 +1,29 @@
 
-The `/inputs` endpoint returns a list of IDs corresponding to Inputs associated to the specific Sender. If this list changes then the associated [IS-04][IS-04] Sender MUST update its version (in registered mode this MUST update the registered resource).
+Non-empty `constraint_sets` array MUST be treated as unordered unless at least one Constraint Set has `urn:x-nmos:cap:meta:preference` attribute.
 
-The `/media-profiles` endpoint allows the client to pass an array of [Media Profiles](Overview.md#media-profile) to a Sender. The sender can then make these restrictions visible to its associated Inputs. The Inputs let the Source and Flow to choose exactly which format from the set they will use. A client can pass a single Media Profile to specify the exact format.
-
-### Role of Media Profiles
-
-The initial state of Media Profiles of a Sender MUST be an empty JSON array `[]`. This state indicated a Sender is unbound by a Media Profile. Creating a connection with such a Sender via [IS-05][IS-05] is not impacted by the Sink Metadata Processing API.
-
-Non-empty array of Media Profiles MUST be treated as unordered unless `preference` property exists.
-
-Once an array of Media Profiles is accepted, a Sender MUST have a Flow and Source which meet at least one Profile. This constraint MUST be adhered to when an external change (i.e. not from the `/media-profiles` endpoint) modifies any aspect of the stream. If the change does not satisfy its Media Profiles, the Sender MUST NOT transmit the signal over the network. The Sender is REQUIRED to take the following actions:
+Once a Sender accepts Constraints, this Sender, the Flow associated with this Sender and the Source associated with this Flow MUST satisfy the Constraints when the Sender is active. At any time if the Source, the Flow and the active Sender cannot satisfy the Constraints, the Sender MUST NOT transmit the Flow over the network. The Sender is REQUIRED to take the following actions:
 
 - MUST set the [IS-05][IS-05] `/active`'s `master_enable` property to `false`.
 - MUST update the [IS-04][IS-04] `subscription`'s `active` property to `false` (regardless of unicast or multicast).
-- MUST respond with `205 Reset Content` when answering `GET /media-profiles` to indicate it is no longer following or adhering to a set. This response SHOULD persist until the next valid `PUT /media-profiles` request.
 
-When a Sender is in this state, ie. responding with a `205 Reset Content`, this indicates a Sender is unbound by a Media Profile and is similar to the initial state.
+Regardless whether a constrained Sender is active, if this Sender, the Flow associated with this Sender or the Source associated with this Flow does not satisfy the Constraints, Sink Metadata Processing API MUST respond with `205 Reset Content` to `GET /constraints` requests. This response SHOULD persist until the Source, the Flow and the Sender satisfy the Constraints.
```

Comment:
Should we use the status endpoint instead of 205 Reset Content?

Reply:
2020/11/01 activity group call: covered in another commit in this PR. Marked as resolved.